gcp_resource_count.sh

```diff
@@ -23,6 +25,7 @@ _make_temp_file() {
 }
 
 PROJECT_ID=""
+MAX_DB_SIZE_GB=1100
```

Comment:
I think the number need to be consistent. Otherwise, people will be confused 
In case of Azure for example, they seems to use mixed between 1024 and 1000 :(

Reply:
What the hell, sure